librosa/core/constantq.py

```diff
@@ -1375,38 +1378,36 @@ def griffinlim_cqt(
         fmin = note_to_hz("C1")
 
     if random_state is None:
-        rng = np.random
+        rng = np.random.default_rng()
     elif isinstance(random_state, int):
-        rng = np.random.RandomState(seed=random_state)
+        rng = np.random.RandomState(seed=random_state)  # type: ignore
     elif isinstance(random_state, np.random.RandomState):
-        rng = random_state
+        rng = random_state  # type: ignore
 
     if momentum > 1:
         warnings.warn(
-            "Griffin-Lim with momentum={} > 1 can be unstable. "
-            "Proceed with caution!".format(momentum),
+            f"Griffin-Lim with momentum={momentum} > 1 can be unstable. "
+            "Proceed with caution!",
             stacklevel=2,
         )
     elif momentum < 0:
-        raise ParameterError(
-            "griffinlim_cqt() called with momentum={} < 0".format(momentum)
-        )
+        raise ParameterError(f"griffinlim_cqt() called with momentum={momentum} < 0")
 
     # using complex64 will keep the result to minimal necessary precision
     angles = np.empty(C.shape, dtype=np.complex64)
     eps = util.tiny(angles)
 
     if init == "random":
         # randomly initialize the phase
-        angles[:] = util.phasor(2 * np.pi * rng.rand(*C.shape))
+        angles[:] = util.phasor(2 * np.pi * rng.random(size=C.shape))  # type: ignore
     elif init is None:
         # Initialize an all ones complex matrix
         angles[:] = 1.0
     else:
         raise ParameterError("init={} must either None or 'random'".format(init))
 
     # And initialize the previous iterate to 0
-    rebuilt = 0.0
+    rebuilt: np.ndarray = np.array(0.0)
```

Comment:
easier to just making this a scalar-shaped array than to deal with types later on